src/pages/Admin/RentedBooksPage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import StandardLayout from 'components/Layout/StandardLayout';
+import RentedBooks from 'components/RentedBook/RentedBooks';
+
+const RentedBooksPage = () => {
+  return (
+    <StandardLayout>
+      <RentedBooks />
+    </StandardLayout>
```

Comment:
I would suggest moving this part in `App.tsx` since you are repeating it in two other components